src/soorgeon/cli.py

```diff
@@ -74,3 +74,21 @@ def refactor(path, log, product_prefix, df_format, single_task, file_format):
 * Jupyter integration: https://ploomber.io/s/jupyter
 * Other editors: https://ploomber.io/s/editors
 """)
+
+
+@cli.command()
+@click.argument("filename", type=click.Path(exists=True))
+@click.option("--deep", "-d", default=None)
+def clean(filename, deep):
+    """
+    Clean a refactored notebook task.
```

Comment:
let's change the description for "Clean a .py or .ipynb file (applies black and isort)"